pkg/util/paramtable/component_param.go

```diff
@@ -1512,6 +1513,16 @@ func (p *queryCoordConfig) init(base *BaseTable) {
 	}
 	p.BalanceIntervalSeconds.Init(base.mgr)
 
+	p.GrowingRowCountWeight = ParamItem{
+		Key:          "queryCoord.growingRowCountWeight",
+		Version:      "2.3.5",
+		DefaultValue: "3.5",
```

Comment:
Better to add the configuration item in milvus.yaml, otherwise, nobody will know the purpose of this configuration.